Emby.Server.Implementations/LiveTv/EmbyTV/RecordingHelper.cs

```diff
@@ -48,12 +48,18 @@ public static string GetRecordingName(TimerInfo info)
 
                 if (!string.IsNullOrWhiteSpace(info.EpisodeTitle))
                 {
+                    var tmpName = name;
                     if (addHyphen)
                     {
-                        name += " -";
+                        tmpName += " -";
+                    }
+                    // Calculate the length of the resulting filename
+                    var recordingNameLength = tmpName.Length + info.EpisodeTitle.Length;
+                    //  Since the filename will be used with file ext. (.mp4, .ts, etc)
+                    if (recordingNameLength < 125) {
```

Comment:
Shouldn't [Encoding.GetByteCount](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getbytecount?view=net-6.0) be used?

Reply:
Totally agree, I'm not really a C# dev, so bare with me here :P. Wrapped the string in ```GetByteCount``` like you suggested and changed the namelength back to 250 https://github.com/jellyfin/jellyfin/pull/7544#discussion_r841227325

Reply:
Should I close this myself, or does the thread opener close it?